packages/utils/package.json

```diff
@@ -16,7 +16,8 @@
   },
   "dependencies": {
     "lodash": "^4.17.19",
-    "logplease": "^1.2.15"
+    "logplease": "^1.2.15",
+    "@origin/utils": "^0.1.0"
```

Comment:
Depends on itself?

Reply:
Oops. Good catch. That was supposed to be in packages/ipfs/package.json. Fixed.